File: day012/day012-2.py
x = int(input("Bir sayı giriniz: "))
temp = 0


for i in range(x):
    temp = i * 2
    print(temp)
    if temp >= x:
        if temp == x:
            print("Sayınız çift bir sayıdır.")
            break
        else:
            print("Sayınız tek bir sayıdır.")
            break

File: day012/day012-2.c
#include <stdio.h>

int main()
{
    
    int x = 6, i, temp = 0;
    
    for (i = 1; i <= x; i++) {
        temp = i*2;
        if (temp >= x ) {
            if (temp > x) {
                printf("Sayınız tek bir sayıdır. ");
                break;
            }
            else {
                printf("Sayınız çift bir sayıdır.");
                break;
            }
        }
        else
            continue;
    }
    return 0;
}
